ci/test.sh

```diff
@@ -57,17 +63,27 @@ function prep_artifacts {
 function run_btests {
     banner "Running baseline tests: zeek"
 
+    if [[ -n "${ZEEK_CI_ZAM}" ]]; then
+        ALTERNATIVE_ARG="-a zam"
+    fi
+
     pushd testing/btest
 
     ZEEK_PROFILER_FILE=$(pwd)/.tmp/script-coverage/XXXXXX \
-        ${BTEST} -z ${ZEEK_CI_BTEST_RETRIES} -d -A -x btest-results.xml -j ${ZEEK_CI_BTEST_JOBS} || result=1
+        ${BTEST} -z ${ZEEK_CI_BTEST_RETRIES} -d -A -x btest-results.xml -j ${ZEEK_CI_BTEST_JOBS} ${ALTERNATIVE_ARG} || result=1
     make coverage
     prep_artifacts
     popd
     return 0
 }
 
 function run_external_btests {
+    # Skip external btests when running against ZAM
+    if [[ -n "${ZEEK_CI_ZAM}" ]]; then
```

Comment:
After looking again, instead ZEEK_CI_ZAM, could also do something like the following and set appropriately in the tasks. Slight preference for that as it seems more generic.
```
ZEEK_CI_SKIP_EXTERNAL_BTESTS
ZEEK_CI_SKIP_UNIT_TESTS
ZEEK_CI_BTEST_EXTRA_ARGS
```

Reply:
Oh yeah, I like that too.